Decisions.TruCap/TruCapRest.cs

```diff
@@ -39,19 +39,17 @@ public static async Task<HttpResponseMessage> TruCapDelete(string url, TruCapAut
 
         try
         {
-            HttpResponseMessage response = await client.SendAsync(request);
+            HttpResponseMessage response = client.Send(request);
             response.EnsureSuccessStatusCode();
             
-            return response;
+            Task<string> resultTask = response.Content.ReadAsStringAsync();
+            resultTask.Wait();
+            
+            return resultTask;
         }
         catch (Exception ex)
         {
```

Comment:
We can remove the try/catch here since we're catching it everywhere we're using it.